### ISSUE TITLE
Compilation fails with rust 1.62

### ISSUE THREAD

**Issue by viraptor** [rbspy/rbspy#368]
Two categories of issues appear with new rust:

unaligned reads:
```
error: reference to packed field is unaligned
    --> ruby-structs/src/ruby_2_5_0.rs:2373:223
     |
2373 | ... () , self . alias_count () , self . complemented_count () , self . body , self . original_id)
     |                                                                 ^^^^^^^^^^^
     |
     = note: `#[deny(unaligned_references)]` on by default
     = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
     = note: for more information, see issue #82523 <https://github.com/rust-lang/rust/issues/82523>
     = note: fields of packed structs are not properly aligned, and creating a misaligned reference is undefined behavior (even if that reference is never dereferenced)
     = help: copy the field contents to a local variable, or replace the reference with a raw pointer and use `read_unaligned`/`write_unaligned` (loads and stores via `*p` must be properly aligned even when using raw pointers)
     = note: this error originates in the macro `$crate::format_args` (in Nightly builds, run with -Z macro-backtrace for more info)
```

and Debug on packed structs:
```
error: `#[derive]` can't be used on a `#[repr(packed)]` struct that does not derive Copy (error E0133)
    --> ruby-structs/src/ruby_2_6_0.rs:7583:10
     |
7583 | #[derive(Debug)]
     |          ^^^^^
     |
     = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
     = note: for more information, see issue #82523 <https://github.com/rust-lang/rust/issues/82523>
     = note: this error originates in the derive macro `Debug` (in Nightly builds, run with -Z macro-backtrace for more info)
```

See full build log https://hydra.nixos.org/build/184359889/nixlog/1

**Comment by acj** [comment]
This should be fixed on main. I don't have an ETA for the next release, but probably soon (~early August). There's been a change to the way rbspy locates and scans binaries/libraries that I want to test more first.

**Comment by ncopa** [comment]
> This should be fixed on main.

Do you know the exact commit that fixes it? We need to either backport the fix or remove rbspy package from alpine as it does not build currently.

**Comment by viraptor** [comment]
@ncopa for NixOS I'm doing this for the fix: https://github.com/NixOS/nixpkgs/pull/182560/files#diff-2b6d9b8a910b526fb7dc8ca327fc303b9dc100c5b5cc985722400907509046b0R27
(commit f5a8eecfbf2ad0b3ff9105115988478fb760d54d)

**Comment by acj** [comment]
Fixed in 0.13.0.
